doc/readme.md

```diff
@@ -0,0 +1,337 @@
+      ```
+      cordova platform add android ios --save
+      ```
+    
+- Add `android` platform with [semver](http://semver.org/) version ^5.0.0 and save it to `config.xml`:   
+```     
+    cordova platform add android@^5.0.0 --save
+```   
+- Add platform by cloning the specified git repo and checkout to the `4.0.0` tag:
+```    
+    cordova platform add https://github.com/myfork/cordova-android.git#4.0.0
+```    
+- Add platform using a local directory named `android`:
+```
+    cordova platform add ../android
+```   
+- Add platform using the specified tarball:
+```
+    cordova platform add ../cordova-android.tgz
+```    
+- Remove `android` platform from the project and from `config.xml`:
+```    
+    cordova platform rm android --save
+``` 
+- List available and installed platforms with version numbers. This is useful to find version numbers when reporting issues:
+``` 
+    cordova platform ls
+```
+- Save versions of all platforms currently added to the project to `config.xml`.
+```
+    cordova platform save
+```
+
+## cordova plugin command
+###Synopsis
+Manage project plugins
+
+### Syntax
+```
+    cordova {plugin | plugins} [ 
+        add <plugin-spec> [..] {--searchpath=<directory> | --noregistry | --link | --save | --browserify} |
+        {remove | rm} {<pluginid> | <name>} --save | 
+        {list | ls} |
+        search [<keyword>] |
+        save |
+    ]
+
+```
+| Sub-command | Option | Description
+|------------------------|-------------|------
+| add `<plugin-spec>` [...] |     | Add specified plugins
+|       |--searchpath `<directory>` | When looking up plugins by ID, look in this directory and each of its subdirectories before hitting the registry. Multiple search paths can be specified. Use ':' as a separator in *nix based systems and ';' for Windows.
+|       |--noregistry             | Don't search the registry for plugins.
+|       |--link                   | When installing from a local path, creates a symbolic link instead of copying files. The extent to which files are linked varies by platform. Useful for plugin development.
+|       |--save                   | Save the `<plugin-spec>` as part of the `plugin` element  into `config.xml`.
+|       |--browserify             | Compile plugin JS at build time using browserify instead of runtime.
+| remove `<pluginid>|<name>` [...] | Remove plugins with the given IDs/name.
+|       |--save                    | Remove the specified plugin from config.xml
+|list                           |  | List currently installed plugins
+|search `[<keyword>]` [...]     |  | Search http://plugins.cordova.io for plugins matching the keywords
+|save                           |  | Save `<plugin-spec` of all plugins currently added to the project                      
+   
+### Plugin-spec
+
+There are a number of ways to specify a plugin:
+```
+    <plugin-spec> : pluginID[@version]|directory|url[#commit-ish]
```

Comment:
According to [this](https://github.com/apache/cordova-lib/blob/master/cordova-lib/src/plugman/fetch.js#L59) we also support fetching plugins from repo subdirectories, so this should look like `pluginID[@version]|directory|url[#commit-ish][:subdirectory]`